corehq/ex-submodules/auditcare/management/commands/gdpr_scrub_user_auditcare.py

```diff
@@ -15,14 +18,12 @@ def add_arguments(self, parser):
         parser.add_argument('username', help="Username to scrub")
 
     def handle(self, username, **options):
+        def update_username(event_dict):
+            audit_doc = NavigationEventAudit.wrap(event_dict)
+            audit_doc.user = new_username
+            event_dict['user'] = new_username
```

Comment:
you don't need this line anymore

Reply:
Also, it might be more efficient to just update the `dict` so you don't have to wrap each of these. (non-blocking since I don't think there are a lot of them, but this is extra work that doesn't need to happen)